language-plutus-core/language-plutus-core.cabal

```diff
@@ -246,14 +276,37 @@ test-suite language-plutus-core-test
         transformers -any,
         tuple -any
 
+test-suite plutus-ir-test
+    type: exitcode-stdio-1.0
```

Comment:
Why merging libraries, but not test suites? Multiple test suites might confuse build tools, IDEs, who knows what.

Reply:
I could merge them. `cabal test` runs both, just fine. I think it's probably helpful to have a focussed test for when you're working on "just" PLC.

Reply:
You could simply create different folders for Plutus Core and Plutus IR inside `test`.

Let me check if those changes break my workflow.

Reply:
That doesn't let you do `cabal test language-plutus-core:plutus-ir-test` to just run the PIR tests, though.

Reply:
Good point. Ok then.

Reply:
(For the record, everything seems to work fine for me)